CHANGES

```diff
@@ -1,5 +1,19 @@
 Version 2.0
 ===========
+Backwards Incompatible Changes
```

Comment:
Preserve the existing convention of an empty line between section title and its content.